mkdocs.yml

```diff
@@ -71,8 +77,12 @@ nav:
     - Outcomes:
       - Overview: philosophy/outcomes/Overview.md
       - Software Delivery Performance: philosophy/outcomes/SoftwareDeliveryPerformance.md
-    - How Pelorus Can Help: philosophy/HowPelorusCanHelp.md
-  - Architecture: Architecture.md
+      - Tech Adoption & ReUse: philosophy/outcomes/platform_adoption.md
+    - philosophy/pelorus.md
+  - Configuration:
```

Comment:
this part of configuration we changed in previous PRs (and will change again soon (we will open an PR to discuss it)), so I believe is best to not touch this part